_includes/functions/matrix-cell-os.md

```diff
@@ -0,0 +1,38 @@
+{% capture /dev/null %}
+
+    {% assign cell_emoji_original = include.state %} <!--- No --->
+
+
+    {% assign android_icon = "<i class='fab fa-android' style='font-size: 24px; color: green;'></i>" %}
+    {% assign linux_icon = "<i class='fab fa-linux' style='font-size: 24px; color: black;'></i>" %}
+    {% assign windows_icon = "<i class='fab fa-windows' style='font-size: 24px; color: blue;'></i>" %}
+    {% assign apple_icon = "<i class='fab fa-apple' style='font-size: 24px; color: black;'></i>" %}
+
+
+    {% assign cell_emoji = cell_emoji_original | replace: "Android", android_icon %}
+    {% assign cell_emoji = cell_emoji | replace: "Linux", linux_icon %}
+    {% assign cell_emoji = cell_emoji | replace: "Windows", windows_icon %}
+
+     <!--- Ensure Apple is only shown once --->
```

Comment:
Build is failing on this line due to mismatching spacing @TorokNRoll

Reply:
I updated this locally (line 16 had an extra space) and pushed to 2024-09-matrix